partner_email_unique/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, _
+from openerp.exceptions import ValidationError
+from openerp.tools import config
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.multi
+    @api.constrains('email')
+    def _check_email(self):
```

Comment:
@robinkeunen For constraints like this, you maybe use that defined in https://github.com/OCA/partner-contact/pull/706

It avoids too much code :smile:

Reply:
Please check my comment at https://github.com/OCA/partner-contact/pull/720#issuecomment-475663856